components/header/header.js

```diff
@@ -0,0 +1,72 @@
+import "../link-button/link-button.js";
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      padding: 20px 0;
+      width: 100%;
+      border-bottom: 1px solid #D3D3D3;
+      background-color: rgba(255, 255, 255, 0.8);
+    }
+
+    header {
+      width: 80%;
+      margin: 0 auto;
+      justify-content: space-between;
+      display: flex;
+      align-items: center;
+    }
+
+    .buttons-container {
+      display: flex;
+    }
+
+    .logo {
+      display: flex;
+      gap: 10px;
+      align-items: center;
+      text-decoration: none;
+    }
+
+    .logo img {
+      width: 45px;
+      height: 22.5px;
+    }
+
+    .logo span {
+      font-size: 20px;
+      font-weight: 500;
+      color: black;
+    }
+
+    .logo img,
+    .logo span {
+    vertical-align: middle;
+    }
+  </style>
+  <header>
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo" />
+        <span>달레 스터디</span>
+      </a>
```

Comment:
@DaleSeo @nhistory @SamTheKorean Have we discussed about what happen after clicking the logo icon?

Reply:
Well, we havent discussed specifically about this part but I mentioned the other parts of the header in #23  so i assumed the designer intended it would work the same as Dale's blog https://www.daleseo.com/.

Reply:
Yep, I agree with #23 and same way with Dale's blog.